Firestore/Source/Public/FIRGetOptions.h

```diff
@@ -65,7 +62,7 @@ typedef NS_ENUM(NSUInteger, FIRGetSource) {
 /**
  * Initializes the get options with the specified source.
  */
-+ (instancetype)optionsWithSource:(FIRGetSource)source NS_SWIFT_NAME(source(_:));
+- (instancetype)optionsWithSource:(FIRGetSource)source NS_SWIFT_NAME(source(_:));
```

Comment:
`optionsWithSource` suggests immutability.  Perhaps this should be `setSource`?

Reply:
If it's an initializer, it should be 

```objc
- (instancetype)initWithSource:(FIRGetSource)source
```

The Swift name will automatically be synthesized to `init(source:)`.